### PR TITLE
Upgrade 12

### PR DESCRIPTION
**Pull request by karesti**
updated to 0.12

### CODE REVIEW

File: src/main/java/org/infinispan/hp/service/DataLoader.java [hunk 1]
```diff
@@ -32,11 +32,9 @@
    public static final String HP_SPELLS_NAME = "spells";
    public static final String HP_MAGIC_NAME = "magic";
 
-   @Inject
```
Comment (geoand) [inline src/main/java/org/infinispan/hp/service/DataLoader.java:35]: :+1:

**Review by geoand — APPROVED** [review]
You might want to wait a couple days since `0.13.0` is scheduled for Wednesday :)

**Comment by karesti** [comment]
@geoand are you allowed to merge? I still need to check for the changes in 0.13

**Comment by karesti** [comment]
@geoand thanks a lot for the review btw!

**Comment by geoand** [comment]
You are very welcome :). Yes I have merge rights, would you like me to do so?

**Comment by karesti** [comment]
@geoand please!

**Comment by geoand** [comment]
Done 😎

**Pull request merged**